packages/flutter/lib/src/widgets/list_wheel_scroll_view.dart

```diff
@@ -0,0 +1,713 @@
+// Copyright 2017 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:math' as math;
+
+import 'package:flutter/foundation.dart';
+import 'package:flutter/rendering.dart';
+
+import 'framework.dart';
+import 'scroll_controller.dart';
+import 'scroll_physics.dart';
+import 'scrollable.dart';
+
+
+/// A box in which widgets on a wheel can be scrolled.
+///
+/// This widget is similar to a [ListView] but with the restriction that all
+/// children must be the same size along the scrolling axis.
+///
+/// The children starts at the middle instead of at the top of the viewport.
+///
+/// Scrolling also ends with the last child at the middle instead of at the
+/// bottom of the viewport.
+///
+/// The children are also rendered as if on a wheel instead of on a plane.
+class ListWheelScrollView extends StatelessWidget {
+  /// Creates a box in which children are scrolled on a wheel.
+  const ListWheelScrollView({
+    Key key,
+    this.controller,
+    this.physics,
+    this.diameterRatio: 2.0,
+    this.perspective: 0.003,
+    @required this.itemExtent,
+    this.clipToSize: true,
+    this.renderChildrenOutsideViewport: false,
+    @required this.children,
+  }) : assert(diameterRatio != null && diameterRatio > 0.0),
+       assert(
+         perspective != null && perspective >= 0.0 && perspective < 0.01,
+         'A perspective too high will be clipped in the z axis and'
+         'un-renderable. Choose a value between 0 and 0.01.'
```

Comment:
missing space between "and" and "un-renderable".

Maybe "will be clipped in the z-axis and therefore not renderable."

Reply:
Replace "Choose a value between" with "Values must be between".

Reply:
(The person reading the error messages may not be the person who wrote the code, so we avoid the second person.)

Reply:
Good point. Done.